src/store/store.ts

```diff
@@ -577,14 +577,23 @@ export const useStore = defineStore('store', {
       )
     },
 
+    /** Whether Amalgamation Information (regular) step is valid. */
+    isAmalgamationInformationRegValid (): boolean {
+      return (
+        this.getAmalgamatingBusinessesValid &&
+        !!this.getCorrectNameOption &&
+        this.getNameTranslationsValid
+      )
```

Comment:
Check (1) table, (2) resulting business name, and (3) name translations (if any).